middleware/logging.go

```diff
@@ -0,0 +1,47 @@
+package middleware
+
+import (
+	"context"
+	"fmt"
+	grpc_logging "github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/logging"
+	"go.uber.org/zap"
+)
+
+// LoggerGRPC adapts zap logger to interceptor logger.
+// Code copied from: https://github.com/grpc-ecosystem/go-grpc-middleware/blob/main/interceptors/logging/examples/zap/example_test.go#L17
```

Comment:
Refrain from including links to a version that may change over time (e.g. `main`, `latest`, major version tags, etc.). Point to the specific version you copied it from instead.

```suggestion
// Code copied from: 
//   https://github.com/grpc-ecosystem/go-grpc-middleware/blob/a18e1e2bacb23afca0f52b228f6b4efbb5f57822/interceptors/logging/examples/zap/example_test.go#L17
```